astacus/common/ipc.py

```diff
@@ -115,6 +120,14 @@ class SnapshotUploadRequest(NodeRequest):
     storage: str
 
 
+class SnapshotUploadRequestV2(SnapshotUploadRequest):
```

Comment:
I would perhaps add some sort of version indicator here - e.g. date when added, or version after which (we're bad at versions, so perhaps date is better). Once long enough time has passed, we can drop the old one and associated code. (I'd say year or two.)